setup.py

```diff
@@ -20,11 +20,11 @@
     install_requires=[
         'docutils',
         'sphinxcontrib-napoleon>=0.5.1',
+        'typing_inspect>=0.3.1',
     ],
     extras_require={
         ':python_version<"3.3"': ['funcsigs'],
         ':python_version<"3.4"': ['enum34'],
-        ':python_version<"3.5"': ['typing'],
```

Comment:
I'd like to keep this as an explicitly listed dependency for the time being, since we import it directly. Other than that, I'm happy to merge this.

Reply:
done